assets/js/index.js

```diff
@@ -0,0 +1,244 @@
+  const initials = document.getElementById("name-initials");
+  if (initials) {
+    const yourScore = {
+      initials: initials.value,
+      score: timer,
+    };
+    storeInLS("feedbackResults", yourScore);
+    initials.value = "";
+  } else {
+    alert("Please enter your initials");
+  }
+};
+//Timer Function called
+const setTimer = () => {
+  const updateTimerValue = () => {
+    //start timer and decrement by 1
+
+    timer -= 1;
+
+    //set text
+    timerSpan.textContent = timer;
+
+    // check if timer is equal to 0
+    if (timer <= 0) {
+      clearInterval(timerId);
+
+      //Delete Timer
+      const removeTimer = document.getElementById("timer-section");
+      removeTimer.remove();
+      gameOver();
+      //Delete question
+      const deleteSection = document.getElementById("question-container");
+
+      if (deleteSection) {
+        deleteSection.remove();
+      }
+    }
+  };
+  const timerId = setInterval(updateTimerValue, 1000);
+};
+
+//Render form
+const renderForm = () => {
+  document.getElementById("timer-section").remove();
+  section = document.createElement("section");
+  section.setAttribute("class", "form-container");
+
+  h6 = document.createElement("h6");
+  h6.setAttribute("class", "timer-score-value");
+  h6.textContent = `your score is ${timer}`;
+
+  const form = document.createElement("form");
+  form.setAttribute("id", "user-score-form");
+
+  const nameInputDiv = document.createElement("div");
+  nameInputDiv.setAttribute("class", "initials-div");
+
+  const textBox = document.createElement("input");
+  textBox.setAttribute("type", "text");
+  textBox.setAttribute("placeholder", "Enter Initials");
+  textBox.setAttribute("id", "name-initials");
+  textBox.setAttribute("class", "name-input");
+  nameInputDiv.append(textBox);
+
+  const buttonDiv = document.createElement("div");
+  buttonDiv.setAttribute("class", "button-input");
+  const button = document.createElement("button");
+  button.setAttribute("type", "submit");
+  button.setAttribute("class", "submit-button");
+  button.setAttribute("class", "submit-score-button");
+  button.textContent = "Submit";
+  buttonDiv.append(button);
+
+  form.append(nameInputDiv, buttonDiv);
+
+  section.append(h6, form);
+
+  main.append(section);
+
+  form.addEventListener("submit", displayResults);
+};
+
+const compareResults = (userAnswer) => {
+  //get user answers
+
+  //check if user's answer matches the correct answer
+
+  if (userAnswer !== questions[questionIndex].correctAnswer) {
+    timer -= 5;
+    if (timer < 0) {
+      timerSpan.textContent = 0;
+    } else {
+      timerSpan.textContent = timer;
+    }
+  }
+};
+
+const selectAnswer = (event) => {
+  //check if answer is selected from one of the options
+  const target = event.target;
+
+  if (target.tagName === "LI") {
+    const userAnswer = target.getAttribute("data-value");
+
+    compareResults(userAnswer);
+  }
+
+  const deleteSection = document.getElementById("question-container");
```

Comment:
Think about the name of this variable, in reference to the other comments further up in this document.